src/api/app/controllers/webui2/package_controller.rb

```diff
@@ -1,5 +1,8 @@
 module Webui2::PackageController
   def webui2_show
+    @tprj, @tpkg = backend_linked_project_package(@package)
+    @tprj = params[:targetproject] if params[:targetproject] # allow to override by parameter
+    @tpkg = params[:targetpackage] if params[:targetpackage] # allow to override by parameter
```

Comment:
With the strong typing hat on: Are we storing in `@tprj` and `@tpkg` information of the same type?

Reply:
Yes, both are strings if they are present

Reply:
ok